frontend/packages/dev-console/integration-tests/features/build-config/shipwright-build/shipwright-build-detail-page.feature

```diff
@@ -0,0 +1,79 @@
+@odc-4981
+Feature: Shipwright build details page
+              As a user, I want check my Shipwright Build and see all related BuildRuns in a second tab.
+
+        Background:
+            Given user has installed OpenShift Pipelines Operator
+              And user is at developer perspective
+              And user has created or selected namespace "aut-shipwright-build-details"
+              And user has installed Shipwright Operator
+              And user is at Add page
+
+
+        @smoke
+        Scenario: Shipwright build page in dev perspective: SWB-01-TC01
+            Given user has created shipwright builds
+             When user navigates to build tab
```

Comment:
It's a navigation item, call we them tabs anyway? 

```suggestion
             When user navigates to "Builds"
```

Reply:
Updated